OneLiners/content.md

```diff
@@ -33,3 +33,10 @@ I went to buy some camouflage trousers the other day but I couldn't find any.
 
 #11
 I had an argument with one of the Seven Dwarves. He wasn’t happy.
+
+#12
+I was walking down the street about 700m above sea level. It was a high street.
+
+#13
+The other day this guy threw a baby goat at me. I said "Are you kidding me!"
```

Comment:
Should this say "guy" or should we use a gender neutral term. We don't want to offend anyone?